scss/variables.scss

```diff
@@ -1,7 +1,9 @@
 // Variables
 $h1FontSize: 2.5em;
 $h1BottomMargin: .5em;
-$greenFontColor: #2D8D37;
+$green2: #2D8D37; 
+$greenFontColor: #72dd7f;
```

Comment:
After comparing the two different greens, I think mine has very little contrast with the background and yours is too bright. I'll try to look for something that's in between.